packages/chats/src/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { Platform } from 'react-native';
 
+export const URI_TON_SURF = 'uri.ton.surf';
```

Comment:
I definitely don't like we have it in constants. This should be removed and passed via React Context instead, IMHO.

Reply:
Yep, I think so

Reply:
@sertony @savelichalex  Done. I also merged onUrlPress and safeURLs into one context for more consistency.